planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -45,192 +45,146 @@
 
   //vehicle param
-  vehicle_param_->width = waitForParam<double>(pnh_, "/vehicle_info/vehicle_width");
-  vehicle_param_->length = waitForParam<double>(pnh_, "/vehicle_info/vehicle_length");
-  vehicle_param_->wheelbase = waitForParam<double>(pnh_, "/vehicle_info/wheel_base");
-  vehicle_param_->rear_overhang = waitForParam<double>(pnh_, "/vehicle_info/rear_overhang");
-  vehicle_param_->front_overhang = waitForParam<double>(pnh_, "/vehicle_info/front_overhang");
+  vehicle_param_->width = declare_parameter<double>("/vehicle_info/vehicle_width", 1.5);
+  vehicle_param_->length = declare_parameter<double>("/vehicle_info/vehicle_length", 2.0);
+  vehicle_param_->wheelbase = declare_parameter<double>("/vehicle_info/wheel_base", 1.0);
+  vehicle_param_->rear_overhang = declare_parameter<double>("/vehicle_info/rear_overhang", 0.5);
+  vehicle_param_->front_overhang = declare_parameter<double>("/vehicle_info/front_overhang", 0.5);
 
   double max_steer_deg = 0;
-  pnh_.param<double>("max_steer_deg", max_steer_deg, 30.0);
+  max_steer_deg = declare_parameter("max_steer_deg", 30.0);
   vehicle_param_->max_steer_rad = max_steer_deg * M_PI / 180.0;
-  pnh_.param<double>("steer_tau", vehicle_param_->steer_tau, 0.1);
+  vehicle_param_->steer_tau = declare_parameter("steer_tau", 0.1);
 
   // mpt param
-  pnh_.param<bool>("is_hard_fix_terminal_point", mpt_param_->is_hard_fix_terminal_point, true);
-  pnh_.param<int>("num_curvature_sampling_points", mpt_param_->num_curvature_sampling_points, 5);
-  pnh_.param<double>("base_point_weight", mpt_param_->base_point_weight, 2000);
-  pnh_.param<double>("top_point_weight", mpt_param_->top_point_weight, 1000);
-  pnh_.param<double>("mid_point_weight", mpt_param_->mid_point_weight, 1000);
-  pnh_.param<double>("lat_error_weight", mpt_param_->lat_error_weight, 10);
-  pnh_.param<double>("yaw_error_weight", mpt_param_->yaw_error_weight, 0);
-  pnh_.param<double>("steer_input_weight", mpt_param_->steer_input_weight, 0.1);
-  pnh_.param<double>("steer_rate_weight", mpt_param_->steer_rate_weight, 100);
-  pnh_.param<double>("steer_acc_weight", mpt_param_->steer_acc_weight, 0.000001);
-  pnh_.param<double>("terminal_lat_error_weight", mpt_param_->terminal_lat_error_weight, 0.0);
-  pnh_.param<double>("terminal_yaw_error_weight", mpt_param_->terminal_yaw_error_weight, 100.0);
-  pnh_.param<double>(
-    "terminal_path_lat_error_weight", mpt_param_->terminal_path_lat_error_weight, 1000.0);
-  pnh_.param<double>(
-    "terminal_path_yaw_error_weight", mpt_param_->terminal_path_yaw_error_weight, 1000.0);
-  pnh_.param<double>("zero_ff_steer_angle", mpt_param_->zero_ff_steer_angle, 0.5);
+  mpt_param_->is_hard_fix_terminal_point = declare_parameter("is_hard_fix_terminal_point", true);
+  mpt_param_->num_curvature_sampling_points = declare_parameter("num_curvature_sampling_points", 5);
+  mpt_param_->base_point_weight = declare_parameter("base_point_weight", 2000);
```

Comment:
please write `2000.0`, not `2000` since it is assumed to be an integer and parameter loading is failed when this node is launched. same for other lines.

Reply:
done in https://github.com/tier4/Pilot.Auto/pull/83/commits/6a9b2564bf86b68712d1ec58dc819154bf9a6dff